pyracf/common/invalid_segment_name_error.py

```diff
@@ -0,0 +1,18 @@
+"""Exception to use when the user passes invalid segment name(s) on an extract request."""
+
+
+class InvalidSegmentNameError(Exception):
```

Comment:
Small naming suggestion. I would maybe suggest renaming this exception to just `InvalidSegmentError`.

Reply:
Sure, change made.